src/app/components/Overview/Overview.tsx

```diff
@@ -1,28 +1,37 @@
-import React, { FunctionComponent, useState } from "react";
+import React, { FunctionComponent } from "react";
 import {
-  Button,
   Card,
   PaginationVariant,
   Toolbar,
   ToolbarContent,
   ToolbarItem,
 } from "@patternfly/react-core";
-import CreateInstance from "@app/Instance/CreateInstance/CreateInstance";
-import { Table, TableColumn } from "@app/components/Table";
-import { Instance } from "../../../types/Instance";
+import { Table, TableColumn, TableRow } from "@app/components/Table";
 import { Pagination } from "@app/components/Pagination/Pagination";
 import { useTranslation } from "react-i18next";
 
-interface InstancesListProps {
-  /** List of columns for the instances table */
-  columnNames: TableColumn[];
-  /** List of instances */
-  instances: Instance[];
+interface OverviewProps {
+  /** List of columns for the table */
+  columns: TableColumn[];
+  /** List of rows for the table */
+  rows: TableRow[];
+  /** Table label */
+  tableLabel: string;
+  /** Custom element you want to be in the toolbar */
+  customToolbarElement?: React.ReactNode;
 }
 
-export const InstancesList: FunctionComponent<InstancesListProps> = ({
-  columnNames,
-  instances,
+/**
+ * The goal of this component is to provide a reusable template composed by:
+ * - Toolbar (with top-pagination)
+ * - Table
+ * - bottom-pagination
+ */
+export const Overview: FunctionComponent<OverviewProps> = ({
```

Comment:
Really minor issue, this component name sounds too much generic to me. What to you think about something like `TableWithPagination` or `PaginatedTable`? Or `EnhancedTable`? Other ideas?